onos-classic/values.yaml

```diff
@@ -54,3 +58,14 @@ ports:
 #     log4j2.appender.console.name = Console
 #     log4j2.appender.console.layout.type = PatternLayout
 #     log4j2.appender.console.layout.pattern = %d{RFC3339} %-5level [%c{1}] %msg%n%throwable
+
+podSecurityContext:
+  runAsUser: 1000
```

Comment:
It is not very clear what this user and group are. May be better to use name or add a comment.

Reply:
no, k8s security context identifies a root user as a user having non-numbers in the id, so even admin or some123 will be considered to be root so we have to use numeric ids